15.1.0 fails to build in TypeScript project (15.0.0 works)

Hi,

when upgrading `contentful/rich-text-html-renderer` and `contentful/rich-text-types` from 15.0.0 to 15.1.0, we get the following error: 

`error TS7016: Could not find a declaration file for module '@contentful/rich-text-html-renderer'. '/[path redacted]/node_modules/@contentful/rich-text-html-renderer/dist/rich-text-html-renderer.es5.js' implicitly has an 'any' type.`

A minimum sample can be found at https://github.com/somehan/2021-08-cf-lib-issue

Steps to reproduce using the sample project:
* Run `npm i`
* Run `npm run build

Expected behavior:
* No output from `tsc`
* Code gets built without errors

Acctual behavior:
* `tsc` reports one error:
 `error TS7016: Could not find a declaration file for module '@contentful/rich-text-html-renderer'. '/home/[path redacted]/cf-lib-issue/node_modules/@contentful/rich-text-html-renderer/dist/rich-text-html-renderer.es5.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/contentful__rich-text-html-renderer` if it exists or add a new declaration (.d.ts) file containing `declare module '@contentful/rich-text-html-renderer';``
  
 When downgrading the libraries to 15.0.0, compilation works as expected.

--- a/packages/rich-text-html-renderer/src/__test__/index.test.ts
+++ b/packages/rich-text-html-renderer/src/__test__/index.test.ts
@@ -1,22 +1,21 @@
 import cloneDeep from 'lodash/cloneDeep';
-import { Document, Block, BLOCKS, MARKS, INLINES } from '@contentful/rich-text-types';
+import { Block, BLOCKS, Document, INLINES, MARKS } from '@contentful/rich-text-types';
 import { documentToHtmlString, Options } from '../index';
 import {
+  embeddedEntryDoc,
+  headingDoc,
   hrDoc,
   hyperlinkDoc,
-  paragraphDoc,
   invalidMarksDoc,
   invalidTypeDoc,
-  headingDoc,
   marksDoc,
-  embeddedEntryDoc,
   olDoc,
-  ulDoc,
+  paragraphDoc,
+  tableDoc,
   quoteDoc,
+  ulDoc,
 } from './documents';
 import inlineEntity from './documents/inline-entity';
-import { tableDoc } from '../../../rich-text-react-renderer/src/__test__/documents';
-import { documentToReactComponents } from '../../../rich-text-react-renderer/src';
 
 describe('documentToHtmlString', () => {
   it('returns empty string when given an empty document', () => {
@@ -255,9 +254,7 @@ describe('documentToHtmlString', () => {
       },
     };
     const document: Document = inlineEntity(asset, INLINES.ASSET_HYPERLINK);
-    const expected = `<p><span>type: ${INLINES.ASSET_HYPERLINK} id: ${
-      asset.target.sys.id
-    }</span></p>`;
+    const expected = `<p><span>type: ${INLINES.ASSET_HYPERLINK} id: ${asset.target.sys.id}</span></p>`;
 
     expect(documentToHtmlString(document)).toEqual(expected);
   });
@@ -273,9 +270,7 @@ describe('documentToHtmlString', () => {
       },
     };
     const document: Document = inlineEntity(entry, INLINES.ENTRY_HYPERLINK);
-    const expected = `<p><span>type: ${INLINES.ENTRY_HYPERLINK} id: ${
-      entry.target.sys.id
-    }</span></p>`;
+    const expected = `<p><span>type: ${INLINES.ENTRY_HYPERLINK} id: ${entry.target.sys.id}</span></p>`;
 
     expect(documentToHtmlString(document)).toEqual(expected);
   });
@@ -291,9 +286,7 @@ describe('documentToHtmlString', () => {
       },
     };
     const document: Document = inlineEntity(entry, INLINES.EMBEDDED_ENTRY);
-    const expected = `<p><span>type: ${INLINES.EMBEDDED_ENTRY} id: ${
-      entry.target.sys.id
-    }</span></p>`;
+    const expected = `<p><span>type: ${INLINES.EMBEDDED_ENTRY} id: ${entry.target.sys.id}</span></p>`;
 
     expect(documentToHtmlString(document)).toEqual(expected);
   });

